v0/destinations/am/transform.js

```diff
@@ -297,6 +297,13 @@ function responseBuilderSimple(
       if (message.isRevenue) {
         // making the revenue payload
         rawPayload = createRevenuePayload(message, rawPayload);
+        // deleting the properties price, product_id, quantity and revenue from evemt_properties since it is already in root
+        if (rawPayload.event_properties) {
+          delete rawPayload.event_properties.price;
+          delete rawPayload.event_properties.product_id;
+          delete rawPayload.event_properties.quantity;
+          delete rawPayload.event_properties.revenue;
+        }
```

Comment:
We should avoid using the `delete` keyword as much as possible. `set(rawPayload, "event_properties", message.properties);` for this method can we filter out these properties when we are setting fields from `property` to `event_properies`

Reply:
We don't have any separate mapping for revenue. It is common for  track call as well. Therefore was deleting those.